lib/llvmopencl/LLVMUtils.h

```diff
@@ -135,4 +135,13 @@ void CloneFunctionIntoAbs(llvm::Function *NewFunc,
 #endif
 }
 
+#ifndef LLVM_OPAQUE_POINTERS
```

Comment:
...then this would always get skipped.

Reply:
These changes are only for the api deprecations,  I _think_ TCE issue is with code generation and would need a different change, likely an arg passed in the build command.  but this problem would exist in the current code, so I am wondering why it hasn't shown up ..